apps/pharos_web/web/core/real_search.ex

```diff
@@ -3,19 +3,40 @@ defmodule Core.RealSearch do
   def execute(%{"topic" => topic, "amount" => amount}) do
     query = %{topic: topic, amount: String.to_integer(amount)}
 
```

Comment:
I agree that we should remove RealSearch in this repo. My suggestion is to create the `Twitter.Search` module in the twitter sub app which can return the hardcoded result) and use this for unit testing the `SearchController`. This means the participants can see the contract when they start quite easily, and the unit test will prob break when they start coding the real thing. Then in the workshop we can demo how to use real/fake for mocking, which they can then apply to their code if they want. 

Sound good?